src/System.Diagnostics.Process/tests/ProcessThreadTests.cs

```diff
@@ -80,16 +79,21 @@ public void TestStartTimeProperty()
                     return;
                 }
 
-                if (ThreadState.Initialized != thread.ThreadState)
+                if (ThreadState.Initialized != thread.ThreadState && ThreadState.Ready != thread.ThreadState)
```

Comment:
if a thread is in ready state, it means it has all the resources but waiting for processor right.. So it's not actually started and starttime might be undefined.